kubernetes/core-svc.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: busybox-service  
+  labels:
+    app: busybox
+spec:
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 8080
```

Comment:
Is this port number right? What does targetPort stands for?

Reply:
Target port is the port on the pod to forward traffic, right? So I guess it should be 80 as well

Reply:
Exactly! TargetPort property refers to port of the container.